docs/course.md

```diff
@@ -129,32 +144,127 @@ The following list of standardized assessments sets is automatically included in
 
 ```json
-    "assessmentSets": [
-        {'abbreviation': 'E', 'name': 'Exam', 'heading': 'Exams', 'color': 'brown1'},
-        {'abbreviation': 'Q', 'name': 'Quiz', 'heading': 'Quizzes', 'color': 'red1'},
-        {'abbreviation': 'PE', 'name': 'Practice Exam', 'heading': 'Practice Exams', 'color': 'yellow1'},
-        {'abbreviation': 'PQ', 'name': 'Practice Quiz', 'heading': 'Practice Quizzes', 'color': 'pink1'},
-        {'abbreviation': 'HW', 'name': 'Homework', 'heading': 'Homeworks', 'color': 'green1'},
-        {'abbreviation': 'P', 'name': 'Prep', 'heading': 'Question Preparation', 'color': 'gray1'},
-        {'abbreviation': 'MP', 'name': 'Machine Problem', 'heading': 'Machine Problems', 'color': 'turquoise1'},
-        {'abbreviation': 'WS', 'name': 'Worksheet', 'heading': 'Worksheets', 'color': 'purple1'}
-    ],
+{
+  "assessmentSets": [
+    {
+      "abbreviation": "E",
+      "name": "Exam",
+      "heading": "Exams",
+      "color": "brown1"
+    },
+    {
+      "abbreviation": "Q",
+      "name": "Quiz",
+      "heading": "Quizzes",
+      "color": "red1"
+    },
+    {
+      "abbreviation": "PE",
+      "name": "Practice Exam",
+      "heading": "Practice Exams",
+      "color": "yellow1"
+    },
+    {
+      "abbreviation": "PQ",
+      "name": "Practice Quiz",
+      "heading": "Practice Quizzes",
+      "color": "pink1"
+    },
+    {
+      "abbreviation": "HW",
+      "name": "Homework",
+      "heading": "Homeworks",
+      "color": "green1"
+    },
+    {
+      "abbreviation": "P",
+      "name": "Prep",
+      "heading": "Question Preparation",
+      "color": "gray1"
+    },
+    {
+      "abbreviation": "MP",
+      "name": "Machine Problem",
+      "heading": "Machine Problems",
+      "color": "turquoise1"
+    },
+    {
+      "abbreviation": "WS",
+      "name": "Worksheet",
+      "heading": "Worksheets",
+      "color": "purple1"
+    }
+  ]
+}
+```
+
+## Assessment units
+
+Each assessment in the course belongs to a _unit_ defined in `infoCourse.json`. Units should be thought of as a unit of instruction, or in PrairieLearn terms, a collection of assessments related to one another, but not necessarily of the same type. This means you can have a unit with two homeworks, one lab, and one exam, for example.
+
+Units are optional and do not affect any behavior by default. Setting the `infoCourse.json` property `"groupBy"` to `"Unit"` (instead of the default `"Set"`) will cause assessments to be grouped together by unit on the student assessments overview page. The order that they are defined in will be the order that they display in, top-down, to students. This can let students view their list of assessments in a chronological order, rather than simply grouped by set.
```

Comment:
groupBy should not be a property of infoCourse.json, at least not only be defined there. I can see newly created courses that want to use "Units" at the start for all courses instances. However, I believe the use case for this property is the most useful at the course instance level.